.github/workflows/TLS-test.yml

```diff
@@ -34,5 +39,11 @@ jobs:
       - name: build
         run: mvn clean package -Dmaven.test.skip=true -B
 
-      - name: test
-        run: mvn test -Dtest=moo -DwildcardSuites=JDBCTLSSuite,TiKVClientTLSSuite,TiSparkTLSSuite -DfailIfNoTests=false
\ No newline at end of file
+      - name: test TLS
+        run: mvn test -am -pl core -Dtest=moo -DwildcardSuites=com.pingcap.tispark.tls.JDBCTLSSuite,com.pingcap.tispark.tls.TiKVClientTLSSuite,com.pingcap.tispark.tls.TiSparkTLSSuite -DfailIfNoTests=false
```

Comment:
maybe `failIfNoTests=true` is better?

Reply:
I'm afraid not.
Because there is no running test in other models, such as tikv-client, you will get the error "_No tests were executed!  (Set -DfailIfNoTests=false to ignore this error._" when setting `failIfNoTests=true`